go.sum

```diff
@@ -35,6 +35,7 @@ github.com/avinetworks/sdk v0.0.0-20200910040148-2ed48a016241/go.mod h1:BcllDeAF
 github.com/avinetworks/sdk v0.0.0-20200910070359-d9ffda19a7dd h1:W+4xqQczmklVpG+JQlY9nBQaSbhbv3MvGaFLX6UvNrY=
 github.com/avinetworks/sdk v0.0.0-20200910070359-d9ffda19a7dd/go.mod h1:BcllDeAFx8PtaMrPxvvuCUo7NRS2x6w+3W17WFDu0sk=
 github.com/avinetworks/sdk v0.0.0-20200910092938-580bed7585cb h1:i6jbYz4bcAN6ropnUUmfG0RCQv+w/SC/MkCXCIUUaAs=
+github.com/avinetworks/sdk v0.0.0-20200921035123-787ceaed92c5 h1:yTqiP3xkjLNhfC7NOkWYz7YxaFhWpGLlSxBJbluJofg=
```

Comment:
Is this bringing in a new SDK version? You don't seem to have any SDK related changes.

Reply:
If not, then can we checkout go.sum and reset it to what it was before this commit and do a push?

Reply:
Sure .. I will revert it back to original.